### ISSUE TITLE
Registration1.html

### ISSUE THREAD

**Issue by pallavikotturu** [pallavikotturu/URPBootstrap#5]
<!DOCTYPE html>
<html>

<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">

<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>

<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>

<head>
    <title></title>
    <meta charset="utf-8" />
</head>
<body style="background-color:#CFE6B4">
    <nav class="navbar navbar-default">
        <div class="container-fluid">
            <div class="navbar-header">
                <a class="navbar-brand" href="Welcome.html">URP Symposium</a>

```
        </div>
        <ul class="nav navbar-nav">
            <li class="active"><a href="StuLogin.html">Student</a></li>
            <li><a href="FacLogin.html">Faculty</a></li>
            <li><a href="Contact.html">Contact</a></li>
        </ul>
    </div>
</nav>


<h1 style="color:#F0832C" align="center"><font size="30">URP Symposium Student Registration</font></h1>

<br />
<br />
<br />
<p style="text-indent: 60px">
    <label style="color:#3D485D">First Name * </label>
    <p style="margin-left: 120px">
    <input type="text"
           id="firstName" />
</p>
```

   <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">Last Name \* </label>
    <p style="margin-left: 120px">
        <input type="text"
               id="lastName" />
    </p>
    <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">TUID \* </label>
    <p style="margin-left: 120px">
        <input type="text"
               id="tuid" />
    </p>
    <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">Email \* </label>
    <p style="margin-left: 120px">
        <input type="text"
               id="email" />
    </p>
    <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">Cell Phone Number \* </label>
    <p style="margin-left: 120px">
        <input type="text"
               id="cellNumber" />
    </p>
    <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">Major \* </label>
    <p style="margin-left: 120px">
        <input type="text"
               id="major" />
    </p>
    <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">Year in School? \* </label>
    <p style="margin-left: 120px">
        <input type="text"
               id="year" />
    </p>
    <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">Have you participated in a previous research poster session? \* </label>
    <p style="margin-left: 120px">
        <input type="radio" name="rdoBefore" id="rdoBefore" value="Yes"> Yes<br>
        <input type="radio" name="rdoBefore" id="rdoBefore" value="No"> No<br>
    </p>
    <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">Semesters that you participated in Research \* </label>
    <p style="margin-left: 120px">
        <input type="text"
               id="semesters" />
    </p>
    <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">Faculty Sponser Name \* </label>
    <p style="margin-left: 120px">
        <input type="text"
               id="facSponser" />
    </p>
    <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">Faculty Sponser Email \* </label>
    <p style="margin-left: 120px">
        <input type="text"
               id="facEmail" />
    </p>
    <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">Poster Title \* </label>
    <p style="margin-left: 120px">
        <input type="text"
               id="posterTtl" />
    </p>
    <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">Poster Description \* </label>
    <p style="margin-left: 120px">
        <input type="text"
               id="posterDescription" />
    </p>
    <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">Time that you can attend the symposium. (3-6 PM is required unless you have class or work) \* </label>
    <p style="margin-left: 120px">
        <input style="color:#3D485D" type="checkbox" name="time" value="3-6 PM">3-6 PM<br>
        <input style="color:#3D485D" type="checkbox" name="time" value="3-5 PM">3-5 PM<br>
        <input style="color:#3D485D" type="checkbox" name="time" value="4-6 PM">4-6 PM<br>
        <input style="color:#3D485D" type="checkbox" name="time" value="5-6 PM">5-6 PM<br />
    </p>
    <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">Expected Graduation Date \* </label>
    <p style="margin-left: 120px">
        <input type="text"
               id="gradDate" />
    </p>
    <br />
    <br />
    <br />
    <p style="margin-left: 120px">
        <button onclick="location.href = 'ThankYouStudent.html';" input type="submit" id="submit" name="Submit" value="Submit" class="btn btn-md btn-default">Submit Proposal</button>

</body>
</html>

**Comment by pallavikotturu** [comment]
<!DOCTYPE html>
<html>





<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">



<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>



<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>

<head>
    <title></title>
    <meta charset="utf-8" />
</head>
<body style="background-color:#CFE6B4">
    

```
<nav class="navbar navbar-default">
    <div class="container-fluid">
        <div class="navbar-header">
            <a class="navbar-brand" href="Welcome.html">URP Symposium</a>

        </div>
        <ul class="nav navbar-nav">
            <li class="active"><a href="StuLogin.html">Student</a></li>
            <li><a href="FacLogin.html">Faculty</a></li>
            <li><a href="Contact.html">Contact</a></li>
        </ul>
    </div>
</nav>
<!--This page is meant for student users who want to enter in a poster proposal, to do just that.
    It allows them to enter information regarding their identity, their research poster proposal, availabilities, and their academic records.
    After a student submits his or her registration entry, they are redirected to a Thank you page within the site which confirms their 
    submission.-->
<h1 style="color:#F0832C" align="center"><font size="30">URP Symposium Student Registration</font></h1>

<br />
<br />
<br />
<p style="text-indent: 60px">
    <label style="color:#3D485D">First Name * </label>
    <p style="margin-left: 120px">
    <input type="text"
           id="firstName" />
</p>
```

   <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">Last Name \* </label>
    <p style="margin-left: 120px">
        <input type="text"
               id="lastName" />
    </p>
    <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">TUID \* </label>
    <p style="margin-left: 120px">
        <input type="text"
               id="tuid" />
    </p>
    <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">Email \* </label>
    <p style="margin-left: 120px">
        <input type="text"
               id="email" />
    </p>
    <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">Cell Phone Number \* </label>
    <p style="margin-left: 120px">
        <input type="text"
               id="cellNumber" />
    </p>
    <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">Major \* </label>
    <p style="margin-left: 120px">
        <input type="text"
               id="major" />
    </p>
    <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">Year in School? \* </label>
    <p style="margin-left: 120px">
        <input type="text"
               id="year" />
    </p>
    <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">Have you participated in a previous research poster session? \* </label>
    <p style="margin-left: 120px">
        <input type="radio" name="rdoBefore" id="rdoBefore" value="Yes"> Yes<br>
        <input type="radio" name="rdoBefore" id="rdoBefore" value="No"> No<br>
    </p>
    <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">Semesters that you participated in Research \* </label>
    <p style="margin-left: 120px">
        <input type="text"
               id="semesters" />
    </p>
    <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">Faculty Sponser Name \* </label>
    <p style="margin-left: 120px">
        <input type="text"
               id="facSponser" />
    </p>
    <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">Faculty Sponser Email \* </label>
    <p style="margin-left: 120px">
        <input type="text"
               id="facEmail" />
    </p>
    <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">Poster Title \* </label>
    <p style="margin-left: 120px">
        <input type="text"
               id="posterTtl" />
    </p>
    <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">Poster Description \* </label>
    <p style="margin-left: 120px">
        <input type="text"
               id="posterDescription" />
    </p>
    <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">Time that you can attend the symposium. (3-6 PM is required unless you have class or work) \* </label>
    <p style="margin-left: 120px">
        <input style="color:#3D485D" type="checkbox" name="time" value="3-6 PM">3-6 PM<br>
        <input style="color:#3D485D" type="checkbox" name="time" value="3-5 PM">3-5 PM<br>
        <input style="color:#3D485D" type="checkbox" name="time" value="4-6 PM">4-6 PM<br>
        <input style="color:#3D485D" type="checkbox" name="time" value="5-6 PM">5-6 PM<br />
    </p>
    <br />
    <p style="text-indent: 60px">
        <label style="color:#3D485D">Expected Graduation Date \* </label>
    <p style="margin-left: 120px">
        <input type="text"
               id="gradDate" />
    </p>
    <br />
    <br />
    <br />
    <p style="margin-left: 120px">
        <button onclick="location.href = 'ThankYouStudent.html';" input type="submit" id="submit" name="Submit" value="Submit" class="btn btn-md btn-default">Submit Proposal</button>

</body>
</html>
